src/app/components/todo-form/todo-form.component.scss

```diff
@@ -1,26 +1,36 @@
-.todo-form{
-    display: flex;
-    justify-content: center;
-    margin-bottom: 20px;
-     .form-content{
-        box-shadow: rgb(38, 57, 77) 0px 20px 30px -10px;
-        .task-input{
-            width: 400px;
-            padding:16px 15px ;
-            border: none;
-            font-size: 20px;
-        
-            &:focus{
-                outline: none;
-            }
-         }
-        .task-input-btn{
-            padding:16px 15px;
-            border: none;
-            font-size: 20px;
-            cursor: pointer;
-        
-         }
- }
+.todo-form {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+  .form-content {
+    box-shadow: rgb(38, 57, 77) 0px 20px 30px -10px;
+    .task-input {
+      width: 400px;
+      padding: 16px 15px;
+      border: none;
+      font-size: 20px;
 
-}
\ No newline at end of file
+      &:focus {
+        outline: none;
+      }
+    }
+    .task-input-btn {
+      padding: 16px 15px;
+      border: none;
+      font-size: 20px;
+      cursor: pointer;
+    }
+  }
+}
+
+.handleErrorMessage {
```

Comment:
You are still using camel case in css selectors which is breaking naming conversion rules.
You should write css selectors in kapap case

Reply:
edit it and show meaningful message on error